### ISSUE TITLE
[SOURCE] links for patch releases lead to 2.7.1

### ISSUE THREAD

**Issue by svekars** [pytorch/pytorch#153733]
### 📚 The doc issue

Here is an example that @stas00 pointed out:

> clicking on [Source] from https://docs.pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html#torch.nn.LayerNorm
> takes me to
> https://github.com/pytorch/pytorch/blob/v2.7.1/torch/nn/modules/normalization.py#L94
> which is 404
> it appears that it prematurely links to 2.7.1 which doesn't yet exist
> it should be:
> https://github.com/pytorch/pytorch/blob/v2.7.0/torch/nn/modules/normalization.py#L94

I checked that it is true for all current 2.7 docs.

**Comment by malfet** [comment]
Removing module regression, as it is not, but rather a bug in a newly added feature...
